Makefile.target

```diff
@@ -135,7 +135,7 @@ obj-y += memory.o savevm.o cputlb.o
 obj-y += memory_mapping.o
 obj-y += dump.o
 obj-$(call lnot,$(CONFIG_ROCKER)) += hw/net/rocker/qmp-norocker.o
-LIBS+=$(libs_softmmu)
+LIBS+=$(libs_softmmu) -lJudy
```

Comment:
Is there something already like libjudy in the qemu build tree?  libjudy looks nice but I can see a road-block trying to get it included for upstream qemu.